core/rawdb/freezer_table_test.go

```diff
@@ -662,87 +662,92 @@ func TestTruncateTail(t *testing.T) {
 	t.Parallel()
 	rm, wm, sg := metrics.NewMeter(), metrics.NewMeter(), metrics.NewGauge()
 	fname := fmt.Sprintf("truncate-tail-%d", rand.Uint64())
+	var maxFileSize uint32 = 50
```

Comment:
Any reason for changing the file size? I don't think it should be changed just for adding a new non-hidden-size check?

Reply:
No, it is just for checking the hidden size. I would change the data sizes instead.